src/style.css

```
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+}
+
+.nav__item {
+  position: relative;
+  font-size: 12px;
+  text-transform: uppercase;
+  margin-left: 20px;
+}
+
+.nav__item:first-of-type {
+  margin-left: 0;
+}
+
+.nav__item:hover {
+  color: #00acdc;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  height: 60px;
+  color: inherit;
+  text-decoration: none;
+}
+
+a.is-active {
+  color: #00acdc;
+}
+
+a.is-active::after {
```

use class selectors instead of tag ones. Or even without if you'd like to use this helper class selector for other cases in a future
```suggestion
.is-active {
  color: #00acdc;
}

.is-active::after {
```